environment.yml

```diff
@@ -7,15 +7,18 @@ channels:
   - defaults
 dependencies:
   - astropy>=5,<6
+  # frictionless 5.8 requires attrs>=22.2,
+  # but a lower version might be installed when the environment is resolved.
+  - attrs>=22.2.0
```

Comment:
If that's the case, then that's a bug in the conda-forge package. Did you report this upstream?

Reply:
seems to be fixed now